apps/vercel/frontend/src/components/config-screen/GettingStartedSection/GettingStartedSection.tsx

```diff
@@ -0,0 +1,21 @@
+import { copies } from '@constants/copies';
+import { Flex, Heading, Paragraph, Box } from '@contentful/f36-components';
+import { styles } from '@locations/ConfigScreen.styles';
+
+export const GettingStartedSection = () => {
+  const { contentPreviewSidebar, contentPreviewSettings, title } =
+    copies.configPage.gettingStartedSection;
+  return (
+    <Box>
+      <Heading className={styles.heading}>{title}</Heading>
+      <Flex>
+        <Paragraph>{contentPreviewSidebar.copy}</Paragraph>
+        <img src={contentPreviewSidebar.src} />
```

Comment:
Awesome catch, thank you! I have added the alt text to these images, and have left as basic `img` tags for now to avoid overcomplicating, but will look at the F36 for future use!